floodviz/views.py

```diff
@@ -34,6 +34,10 @@ def timeseries_data():
     # Hydrodata data clean and write
     j = hydrograph_utils.req_hydrodata(sites, hydro_start_date, hydro_end_date, url_nwis_prefix)
     timeseries_data = hydrograph_utils.parse_hydrodata(j)
+
+    # Write data to file for post build thumbnail
+    with open('floodviz/static/data/hydrograph_data.json', 'w') as f:
```

Comment:
I'd also like this to happen only when we ask it to be generated. maybe through a config.py variable